src/main/java/com/speaker/service/MessageServiceImpl.java

```diff
@@ -1,24 +1,51 @@
 package com.speaker.service;
 
+import com.speaker.convertors.MessageConvertor;
+import com.speaker.dto.MessageDTO;
 import com.speaker.entities.Message;
+import com.speaker.repository.AccountRepository;
 import com.speaker.repository.MessageRepository;
+import com.speaker.service.util.Pair;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.Optional;
+
+import static com.speaker.service.util.StringParser.parseOptionalStringBySpace;
 
 @Service
 @RequiredArgsConstructor
 public class MessageServiceImpl implements MessageService {
     private final MessageRepository messageRepository;
+    private final AccountRepository accountRepository;
+    private final MessageConvertor messageConvertor;
 
     @Override
     public List<Message> getMessagesByAccountId(int accountId) {
         return messageRepository.findAllByAccountId(accountId);
     }
 
     @Override
-    public int create(Message message) {
-        return messageRepository.insert(message);
+    public Response addMessage(List<MessageDTO> messageDTO) {
+        for (MessageDTO message : messageDTO) {
```

Comment:
lets will use stream

Reply:
Comments accepted, corrections made. Thank you